CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+
+A changelog is kept per crate, see:
+
+- [bitcoin CHANGELOG](bitcoin/CHANGELOG.md)
+
+- [addresses CHANGELOG](addresses/CHANGELOG.md)
+- [base58 CHANGELOG](base58/CHANGELOG.md)
+- [hashes CHANGELOG](hashes/CHANGELOG.md)
+- [internals CHANGELOG](internals/CHANGELOG.md)
+- [io CHANGELOG](io/CHANGELOG.md)
+- [primitives CHANGELOG](primitives/CHANGELOG.md)
+- [units CHANGELOG](units/CHANGELOG.md)
```

Comment:
Maybe put them in backticks. I find it really useful to find out that things are "code"-related.
```suggestion
- [`bitcoin` CHANGELOG](bitcoin/CHANGELOG.md)
- [`addresses` CHANGELOG](addresses/CHANGELOG.md)
- [`base58` CHANGELOG](base58/CHANGELOG.md)
- [`hashes` CHANGELOG](hashes/CHANGELOG.md)
- [`internals` CHANGELOG](internals/CHANGELOG.md)
- [`io` CHANGELOG](io/CHANGELOG.md)
- [`primitives` CHANGELOG](primitives/CHANGELOG.md)
- [`units` CHANGELOG](units/CHANGELOG.md)
```